executor/coprocessor.go

```diff
@@ -70,12 +68,66 @@ func (h *CoprocessorDAGHandler) HandleRequest(ctx context.Context, req *coproces
 		if chk.NumRows() == 0 {
 			break
 		}
-		err = h.appendChunk(chk, tps)
+		partChunks, err = h.buildChunk(chk, tps)
 		if err != nil {
 			break
```

Comment:
How about `return h.buildErrorResponse(err)` instead of `break`?

Reply:
great, done.